dev/com.ibm.ws.classloading/resources/OSGI-INF/l10n/classloader.properties

```diff
@@ -21,14 +21,19 @@ classloader.service.desc=Defines the settings for an application classloader.
 #attributes for the classloader type
 #delegation
 classloader.delegation=Delegation
-classloader.delegation.desc=Controls whether parent classloader is used before or after this classloader. If parent first is selected then delegate to immediate parent before searching the classpath. If parent last is selected then search the classpath before delegating to the immediate parent.
+classloader.delegation.desc=Controls whether parent classloader is used before or after this class loader. If parent first is selected then delegate to immediate parent before searching the classpath. If parent last is selected then search the classpath before delegating to the immediate parent.
 # Option descriptions are not supported in Metatype spec but we may extend for that
 parentFirst=Parent first
 parentLast=Parent last
-# libraryRefs
+# Private libraryRefs
 classloader.library.ref=Library
 classloader.library.ref$Ref=Library references
-classloader.library.ref.desc=List of library references. Library class instances are unique to this classloader, independent of class instances from other classloaders.
+classloader.library.ref.desc=List of library references. Library class instances are unique to this class loader, independent of class instances from other classloaders. The referenced library class path is searched after searching the class path of this class loader.
```

Comment:
For: `Library class instances are unique to this class loader, independent of class instances from other classloaders`

`classloaders` should be separated to `class loader`